package.json

```diff
@@ -4,18 +4,27 @@
     "description": "This repository is made by students of Code Chrysalis",
     "main": "index.js",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start:dev": "webpack && node server.js",
+        "webpack": "webpack --watch"
```

Comment:
This could also use some reorganizing. For a webpack + node server setup, here is what you usually find:

* build - builds the project for production
* start - starts the node server (no building)
* dev - starts server (nodemon) and webpack in watch mode, complex projects support hot reloading here, but might be too tricky for this one given the remaining time.
* postinstall - triggers build after installing dependencies, this way we don't need to check in bundle.js